runtime/lua/vim/ui/img/utils/size.lua

```diff
@@ -0,0 +1,83 @@
+---@class vim.ui.img.utils.Size
```

Comment:
all of these `img/utils/*` should live in `img/*` with underscore. don't need extra nesting of `img/utils`.